source/localizable/routing/loading-and-error-substates.md

```diff
@@ -152,6 +152,17 @@ that substate (without updating the URL). The "reason" for the error
 (i.e. the exception thrown or the promise reject value) will be passed
 to that error state as its `model`.
 
+The model hooks (`beforeModel`, `model`, and `afterModel`) of an error substate
+are NOT called. Only the `setupController` method of the error substate is
+called with the `error` as the model. See example below:
+
+```js
+setupController: function(controller, error) {
+  Ember.Logger.debug(error.message);
+  this._super(controller, error);
```

Comment:
@locks was thinking that using `(controller, error)` helps clarify that it's an error not a model object.